scripts/bugtool-plugin/xapi.xml

```diff
@@ -1 +1 @@
-<capability pii="yes" max_size="16384" max_time="60" mime="text/plain" checked="true"/>
+<capability pii="yes" max_size="1638400" max_time="60" mime="text/plain" checked="true"/>
```

Comment:
It looks like this limit applies to all files in the directory together, not individual files: https://github.com/xenserver/status-report/blob/master/xen-bugtool#L517
So this limit is not enough for the up to 100MB of files in /var/log/dt. We either need to increase this value further, or create a new capability of max_size=100MB only for /var/log/dt